.github/workflows/deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Test
+    branches:
+      - main
+      - development
+    types:
+      - completed
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: ${{ github.event.workflow_run.head_branch || github.ref }}
+
+      - name: Set BRANCH_TAG
+        uses: nimblehq/branch-tag-action@v1.2
+        with:
+          ref: ${{ github.event.workflow_run.head_branch || github.ref }}
+
+      - name: Set HEROKU_APP_NAME
+        run: |
+          if [[ $BRANCH_TAG = "latest" ]]
+          then
+            echo "HEROKU_APP_NAME=${{ secrets.HEROKU_APP_NAME }}" >> $GITHUB_ENV
+          else
+            echo "HEROKU_APP_NAME=${{ secrets.HEROKU_APP_NAME }}-staging" >> $GITHUB_ENV
+          fi
+
+      - uses: akhileshns/heroku-deploy@v3.7.8
+        with:
+          heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
+          heroku_app_name: $HEROKU_APP_NAME
+          heroku_email: ${{ secrets.HEROKU_APP_NAME }}
+          usedocker: true
+          env_file: ".env"
```

Comment:
While this `.env` is committed to the repository, here might be future env variables that should remain private. So think of using Github secrets to append these environment variables to an `.env` file as done on the website:

![image](https://user-images.githubusercontent.com/696529/102972272-5eb99e80-452d-11eb-89e4-dbd7916c3568.png)

Reply:
@olivierobert Noted